Changelog.md

```diff
@@ -8,6 +8,7 @@ Compiler Features:
  * SMTChecker: Support inherited state variables.
  * SMTChecker: Support tuples and function calls with multiple return values.
  * SMTChecker: Support ``delete``.
+ * Metadata: Add another field encoding the compiler version to the CBOR metadata.
```

Comment:
Hm, the metadata is the json whose hash is part of the code - this is something we probably do not have a name for yet...

Reply:
Perhaps just "Assembler: encode the compiler version in the deployed bytecode"?

Reply:
In ``0.4.16`` we have ``Metadata: Store experimental flag in metadata CBOR.`` :-). But I'll change it anyways!